### ISSUE TITLE
Toggling between dark and light theme visually breaks calendar

### ISSUE THREAD

**Issue by forivall** [dendronhq/dendron#3633]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [ ] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [x] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [ ] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

After changing themes from light to dark o

r dark to light, (`workbench.action.toggleLightDarkThemes`) the calendar has a broken visual state

## To Reproduce

1. Open the calendar view
2. Use the "workbench.action.toggleLightDarkThemes" from the quick action menu
3. Observe the calendar
4. Use "reload window" to fix it

## Expected behavior

The Calendar should appear as it does on a fresh load of the extension

## Screenshots

Light to Dark: ![Light to Dark](https://user-images.githubusercontent.com/760204/194183035-29ff10f8-e829-4897-a807-5316f397b4a4.png)

Dark to Light: ![Dark to Light](https://user-images.githubusercontent.com/760204/194183113-9997ad06-8f90-4fae-b9f2-4f62bd6042e1.png)

## Desktop (please complete the following information)

- OS: macOS 12.6
- Browser (if with published websites; include version): [e.g. chrome, safari]
- Versions (all that apply to bug):
  - Dendron Extension: v0.114.0
  - VS Code version: 1.71.2
  - Dendron CLI: N/A

## Dendron Log file

Please attach the output of `> Dendron:Dev: Open Logs` here

_Will do if requested. I took a look and I doubt it will be useful, but I can provide if desired_

**Comment by hikchoi** [comment]
Thank you for the report.
I can't reproduce this exactly like you described, but it breaks in a different way on my end.
We'll let you know when this gets fixed!

pinging @namjul for visibility.

**Comment by forivall** [comment]
> I can't reproduce this exactly like you described, but it breaks in a different way on my end.

Yeah, I have a complex vscode configuration, so I wouldn't expect other replications to be exact (and I didn't want to go through the effort of setting up a pristine replication unless it was necessary)

Thanks!
